.github/workflows/docker.yml

```diff
@@ -1,6 +1,6 @@
 name: Build & Push
 # Build & Push builds the simapp docker image on every push to main and
-# and pushes the image to https://hub.docker.com/r/interchainio/simapp/tags
+# and pushes the image to https://github.com/cosmos/cosmos-sdk/pkgs/container/simapp
```

Comment:
Shouldn't this be a ghcr.io url?

Reply:
I wanted to keep the same "behavior" as before, where we link to the website that displays the tag list.
Turns out `https://ghcr.io/cosmos/simapp` redirects to https://github.com/cosmos/cosmos-sdk/pkgs/container/simapp as well, so I will update.